scripts/maint/updateFallbackDirs.py

```diff
@@ -2244,7 +2253,11 @@ def list_fallbacks(whitelist, exact=False):
   """ Fetches required onionoo documents and evaluates the
       fallback directory criteria for each of the relays,
       passing exact to apply_filter_lists(). """
-  print "/* type=fallback */"
+  if whitelist['check_existing']:
+      print "/* type=fallback */"
+  else:
+      print "/* type=whitelist */"
```

Comment:
This is the type of the output list, not the input list. The output list always contains fallbacks, regardless of the type of the input list.

(More precisely: there's a mode that generates potential fallback candidates. But I don't think we've used it for a while. If we ever start using it again, we could update the type for that mode. If we don't use it for another year or two, we'll take it out instead.)

Also, this change will break existing parsers:
https://gitweb.torproject.org/torspec.git/tree/dir-list-spec.txt#n140

Reply:
Undid this change in 3d5af708569fe83f1829395dd28b6b67cf435243.